packages/gitgraph-react/src/stories/index.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { Gitgraph, Branch } from "../Gitgraph";
+
+import { storiesOf } from "@storybook/react";
+
+class GitgraphPlayground extends React.Component<any, any> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      branches: [],
+    };
+  }
+
+  public addCommit = (branch?: Branch) => {
+    if (branch) {
+      branch.commit(this.state[`commitMessage${branch.name}`]);
+    } else {
+      this.state.gitgraph.commit(this.state.commitMessage);
+    }
+  }
+
+  public addBranch = () => {
+    this.setState((state: any) => ({
+      branches: [...state.branches, this.state.gitgraph.branch(this.state.branchName)],
+    }));
+  }
+
+  public handleChange = (name: string) => (e: React.SyntheticEvent<HTMLInputElement>): void => {
+    this.setState({ [name]: e.currentTarget.value });
+  }
+
+  public render() {
+    return (
+      <div>
+        <form onSubmit={(e) => e.preventDefault() || this.addCommit()}>
```

Comment:
For now it's light and ugly, but I will maybe add semantic-ui to have a better playground